config_child_item.go

```diff
@@ -0,0 +1,7 @@
+package libyaml
+
+// ConfigChildItem represents a subitem.  This is normally found in select_one and select_many types.
+type ConfigChildItem struct {
+	Name  string `yaml:"name" json:"name" validate:"required"`
+	Title string `yaml:"title" json:"title"`
```

Comment:
i do see this comment https://github.com/replicatedcom/replicated-config-render/blob/master/src/components/ConfigFieldSelectMany.js#L20
```
// TODO: does required make sense here?
```